templates/components/category/shop-by-price.html

```diff
@@ -1,4 +1,5 @@
 {{#if theme_settings.shop_by_price_visibility}}
+  {{#if shop_by_price}}
```

Comment:
you could consider combining this into one check which would be a little cleaner:

`{{#and theme_settings.shop_by_price_visibility shop_by_price}}`

Reply:
Done...